Mathlib/Algebra/BigOperators/Basic.lean

```diff
@@ -215,58 +215,11 @@ theorem map_prod [CommMonoid β] [CommMonoid γ] {G : Type*} [MonoidHomClass G 
 #align map_prod map_prod
 #align map_sum map_sum
 
-section Deprecated
-
 #align monoid_hom.map_prod map_prodₓ
 #align add_monoid_hom.map_sum map_sumₓ
 #align mul_equiv.map_prod map_prodₓ
 #align add_equiv.map_sum map_sumₓ
 
-@[deprecated _root_.map_list_prod]
-protected theorem RingHom.map_list_prod [Semiring β] [Semiring γ] (f : β →+* γ) (l : List β) :
-    f l.prod = (l.map f).prod :=
-  map_list_prod f l
-#align ring_hom.map_list_prod RingHom.map_list_prod
```

Comment:
IMHO, we should realign these lemmas to the generic lemmas in the root namespace.